tests/benchmarks/complete_schema.py

```diff
@@ -1,7 +1,9 @@
 from decimal import Decimal
 
+from pydantic_core import CoreSchema
```

Comment:
Probably best to back out the type hint changes from this file; as it is run from under cargo it'll be a bit of a faff to configure to run under the right virtual environment.

(PyO3 needs to get better at this!)